spec/services/client_secrets_loader_spec.rb

```diff
@@ -0,0 +1,24 @@
+describe ClientSecretsLoader do
+  context "everything's fine" do
+    it "loads the secrets" do
+      ENV["google_client_id"] = "totaly_real_client_id"
```

Comment:
A trick with ENV vars and tests is to always reset them after the test otherwise you pollute the rest of your test suite.
This can often lead to unintentional fails and passes.
https://thoughtbot.com/blog/testing-and-environment-variables#override-environment-variables-on-a-per-test-basis

Reply:
yeah found this out the hard way! Is deleting them on lines 8 and 9 not enough? is there something I'm missing here?

Reply:
Sorry my bad, I missed the deleting bits.
It might be clearer to delete and then set the ENV vars in a before block.

Reply:
👍